src/payment/strategies/googlepay/googlepay-payment-strategy.ts

```diff
@@ -217,4 +196,39 @@ export default class GooglePayPaymentStrategy implements PaymentStrategy {
             this._store.dispatch(this._paymentMethodActionCreator.loadPaymentMethod(methodId)),
         ]);
     }
+
+    private _buttonClickEvent(methodId: string): (event: Event) => Promise<InternalCheckoutSelectors> {
```

Comment:
this is a method name, so it should contain a verb, maybe `_handleButtlonClickedEvent` ?